modules/specialized-guides/pages/public-cloud-guide/overview.adoc

```diff
@@ -10,7 +10,8 @@ These procedures have been tested on {awslongform}, {azure}, and {gcplongform}.
 
 [NOTE]
 ====
-{payglongform} or {payg} services currently work solely on AWS. Solutions are in development for both {azure} and {gcplongform}.
+{payglongform} or {payg} services currently work on both {awslongform} and {azure}.
+Solutions are also in development for {gcplongform}.
```

Comment:
Should we add this in the documentation? I don't think it's a good idea to "promise" something like this. The documentation is instructions, any other type of communication goes via other product channels.

Reply:
To discuss it, and change it I created https://github.com/SUSE/spacewalk/issues/23547